carbon/app.go

```diff
@@ -510,6 +510,12 @@ func (app *App) Start(version string) (err error) {
 		carbonserver.SetNoServiceWhenIndexIsNotReady(conf.Carbonserver.NoServiceWhenIndexIsNotReady)
 		carbonserver.SetRenderTraceLoggingEnabled(conf.Carbonserver.RenderTraceLoggingEnabled)
 
+		if conf.Carbonserver.MaxStreamingChannelSize == nil || *conf.Carbonserver.MaxStreamingChannelSize < 0 {
+			carbonserver.SetMaxStreamingChannelSize(100)
```

Comment:
Default configs are in carbon/config.go
I think it make sense to move defaults there

Reply:
We had this discussion and decided to remove the whole parameter.